Closure_Front_End/src/components/LineChart.vue

```diff
@@ -0,0 +1,44 @@
+
+
+<script>
+import { defineComponent } from "vue";
+import { Line } from "vue3-chart-v2";
+
+export default defineComponent({
+  name: "MonthlyChart",
+  extends: Line,
+  // props: {
+  //   chartData: {
+  //     type: Object,
+  //     required: true
+  //   },
+  //   chartOptions: {
+  //     type: Object,
+  //     required: false
+  //   },
+  // },
+  mounted() {
+    this.renderChart({
+      labels: [0,1, 2, 3, 4, 5, 6, 7, 8],
+      datasets: [
+        {
+          label: "total points",
+          backgroundColor: "#999999",
+          fill: false,
+          data: [0,25, 56, 70, 80, 90, 105, 125, 134],

```

Comment:
`134`? What's this?

Reply:
the number of points required for completing a CS degree.

Reply:
1. Does the order mean anything? 
2. is it hardcoded for development purposes?

Reply:
1. no, it approximates the cummulative number of points up to the appropriate semseter
2. it is hardcoded

Reply:
2. why?